pybleau/app/io/tests/test_plotting_roundtrip.py

```diff
@@ -32,51 +32,54 @@ def setUp(self):
                        "plot", "plot_factory", "contained_plot_map",
                        "inspectors")
 
-    def test_round_trip_df_plotter_basic_types(self):
+    def test_round_trip_df_plotter_with_hist(self):
         config = HistogramPlotConfigurator(data_source=TEST_DF)
         config.x_col_name = "Col_4"
         desc = PlotDescriptor(x_col_name="Col_4", plot_config=config,
                               plot_title="Plot 1")
+        self.assert_df_plotter_roundtrip(desc)
 
+    def test_round_trip_df_plotter_with_bar(self):
         config2 = BarPlotConfigurator(data_source=TEST_DF)
         config2.x_col_name = "Col_1"
         config2.y_col_name = "Col_2"
         desc2 = PlotDescriptor(x_col_name="Col_1", y_col_name="Col_2",
                                plot_config=config2, plot_title="Plot 2")
+        self.assert_df_plotter_roundtrip(desc2)
 
+    def test_round_trip_df_plotter_with_scatter(self):
         config3 = ScatterPlotConfigurator(data_source=TEST_DF)
         config3.x_col_name = "Col_1"
         config3.y_col_name = "Col_2"
         desc3 = PlotDescriptor(x_col_name="Col_1", y_col_name="Col_2",
                                plot_config=config3, plot_title="Plot 3")
+        self.assert_df_plotter_roundtrip(desc3)
 
+    def test_round_trip_df_plotter_with_colored_scatter(self):
         config4 = ScatterPlotConfigurator(data_source=TEST_DF)
         config4.x_col_name = "Col_1"
         config4.y_col_name = "Col_2"
         config4.z_col_name = "Col_3"
         desc4 = PlotDescriptor(x_col_name="Col_1", y_col_name="Col_2",
                                z_col_name="Col_3",
                                plot_config=config4, plot_title="Plot 4")
+        self.assert_df_plotter_roundtrip(desc4)
 
+    def test_round_trip_df_plotter_with_line_plot(self):
         config5 = LinePlotConfigurator(data_source=TEST_DF)
         config5.x_col_name = "Col_1"
         config5.y_col_name = "Col_2"
         desc5 = PlotDescriptor(x_col_name="Col_1", y_col_name="Col_2",
                                plot_config=config5, plot_title="Plot 5")
+        self.assert_df_plotter_roundtrip(desc5)
 
+    def test_round_trip_df_plotter_bar(self):
```

Comment:
Seems like a duplicate of line 45. Are both necessary?

Reply:
Different because it's a serialization of a different type of plot.